### ISSUE TITLE
Accessibility Server doc page is missing

### ISSUE THREAD

**Issue by ScatteredComet** [godotengine/godot-docs#11824]
**Your Godot version:** 4.6 / latest

**Issue description:**

On both the stable and latest docs, `AccessibilityServer`'s page doesn't exist e.g. the following leads nowhere:

https://docs.godotengine.org/en/latest/classes/class_accessibilityserver.html

However there is an `*.xml` file that was committed in the [PR that created the class](https://github.com/godotengine/godot/pull/116839). So I guess this is some kind of doc generation bug? There is no generated `*.rst` file in the classes/ directory for it.

The `*.xml` is also missing an enum:

```
enum AccessibilityMode {
	ACCESSIBILITY_AUTO,
	ACCESSIBILITY_ALWAYS,
	ACCESSIBILITY_DISABLED,
};
```

**URL to the documentation page:**

https://docs.godotengine.org/en/latest/classes/class_accessibilityserver.html
https://docs.godotengine.org/en/stable/classes/class_accessibilityserver.html

**Comment by ScatteredComet** [comment]
After writing some "terrible python code" (as Matt Parker would say), I believe this is the only doc `*.xml` that hasn't been converted to an `*.rst`

**Comment by AThousandShips** [comment]
The class reference hasn't been synced yet it's just been 4 days, the last one was last week:
* https://github.com/godotengine/godot-docs/pull/11798

A new one will happen tomorrow, they happen at night CET on Saturdays
